front/src/modules/ui/theme/hooks/useColorScheme.ts

```diff
@@ -2,20 +2,38 @@ import { useCallback } from 'react';
 import { useRecoilState } from 'recoil';
 
 import { currentUserState } from '@/auth/states/currentUserState';
-import { ColorScheme, useUpdateUserMutation } from '~/generated/graphql';
+import {
+  ColorScheme,
+  UserSettings,
+  useUpdateOneWorkspaceMemberMutation,
+  useUpdateUserMutation,
+  WorkspaceMember,
+} from '~/generated/graphql';
 
 export function useColorScheme() {
   const [currentUser, setCurrentUser] = useRecoilState(currentUserState);
 
   const [updateUser] = useUpdateUserMutation();
+  const [updateWorkspaceMember] = useUpdateOneWorkspaceMemberMutation();
 
-  const colorScheme = !currentUser?.settings?.colorScheme
-    ? ColorScheme.System
-    : currentUser.settings.colorScheme;
+  const colorScheme =
+    !currentUser?.workspaceMember.settings?.colorScheme &&
+    !currentUser?.settings?.colorScheme
+      ? ColorScheme.System
```

Comment:
Read colorScheme from `workspaceMember` before `user`. Settings are connected to both, so we can update either.